samples/dotnet/TestPalmApi/Program.cs

```diff
@@ -0,0 +1,162 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using Connectors.AI.PaLM.Skills;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.Memory;
+using Microsoft.SemanticKernel.Orchestration;
+using RepoUtils;
+
+namespace TestPalmApi;
+
+internal class Program
+{
+    //"[PUT YOUR PALM API KEY HERE]"; - please delete the key below after test ;D
+    const string PALM_API_KEY = "AIzaSyBckgGXO_DcMSmoRnE7UHGTKvfO7GuDU_8";
+    const string Model = "text-bison-001";
+    const string ModelEmbedding = "embedding-gecko-001";
+    const string ModelToken = "chat-bison-001";
+    static async Task Main(string[] args)
+    {
+        //run text embedding
+        await RunEmbedAsync();
```

Comment:
There's already a project for examples called KernelSyntaxExamples. Please move the code there.

Reply:
moved ok